### ISSUE TITLE
Build fails because SDL2 is not found

### ISSUE THREAD

**Issue by Stokestack** [google/monster-mash#9]
Running CMake on the src directory produces this output (on Mac OS):

```
CMake Error at CMakeLists.txt:85 (find_package):
  By not providing "FindSDL2.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "SDL2", but
  CMake did not find one.

  Could not find a package configuration file provided by "SDL2" with any of
  the following names:

    SDL2Config.cmake
    sdl2-config.cmake
```

**Comment by luoyun7123** [comment]
emmm，the provider believes that most people have SDL2 and opengl in their computers, if you don't have them, just install one ......

**Comment by Stokestack** [comment]
Ummm... nope.

**Comment by micwoj92** [comment]
Project should have all dependencies listed, what if I want to build it on a server without gui? Doubt I will have opengl and sdl2 there.

**Comment by albertotono** [comment]
Yes, please make sure you Install SDL2, if you are on Linux you can follow [these steps:](https://lazyfoo.net/tutorials/SDL/01_hello_SDL/linux/index.php) 

(The issue is reported [here](https://stackoverflow.com/questions/54005991/what-happened-to-findsdl2-in-cmake)

**Comment by tanxianjun2016** [comment]
mac os , already install SDL2, cmake note:
/Library/Frameworks/SDL2.framework/Headers/SDL_opengles2.h:35:10: fatal error: 'GLES2/gl2platform.h' file not found
#include <GLES2/gl2platform.h>
